.github/workflows/tekton.yaml

```diff
@@ -27,18 +27,32 @@ name: Checks
 
 jobs:
   Tekton:
-
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v3
 
       - name: setup kind cluster
         uses: container-tools/kind-action@v2
 
-      - name: Run ec task
+      - name: install tkn cli
```

Comment:
Found [this action](https://github.com/marketplace/actions/install-tkn), which does basically the same as here. I do like having versions managed in a single place, perhaps in a followup we can see about using the [tools approach](https://github.com/hacbs-contract/ec-cli/blob/9b3412a103f0ec3b70cb4ceaeb4750ffdcac85e8/internal/tools/tools.go) and version via `go.mod`.